ddmrp/models/stock_buffer.py

```diff
@@ -743,12 +743,17 @@ def _get_product_sellers(self):
         if not sellers:
             # fallback to all sellers
             sellers = all_sellers
-        # When the current transaction changed the sequence, it may happen that
-        # the sellers' recordset is not correctly sorted by default.
-        sellers = sellers.sorted(key="sequence")
```

Comment:
According to the comment, then we better do something like:
```
sellers = sellers.sorted(key=lambda s: (s.sequence, -s.min_qty, price))
```
to comply with `_order = 'sequence, min_qty desc, price'`